catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/transformers/sierra/SierraItems.scala

```diff
@@ -29,27 +29,31 @@ trait SierraItems extends Logging with SierraLocation {
       .flatten
   }
 
-  def transformItemData(sierraItemData: SierraItemData): UnidentifiedItem = {
+  def transformItemData(sierraItemData: SierraItemData): Option[UnidentifiedItem] = {
     info(s"Attempting to transform ${sierraItemData.id}")
-    UnidentifiedItem(
-      sourceIdentifier = SourceIdentifier(
-        IdentifierSchemes.sierraSystemNumber,
-        sierraItemData.id
-      ),
-      identifiers = List(
-        SourceIdentifier(
-          identifierScheme = IdentifierSchemes.sierraSystemNumber,
+    if (sierraItemData.deleted) {
+      None
+    } else {
+      Some(UnidentifiedItem(
+        sourceIdentifier = SourceIdentifier(
+          IdentifierSchemes.sierraSystemNumber,
           sierraItemData.id
-        )
-      ),
-      locations = getLocation(sierraItemData).toList,
-      visible = !sierraItemData.deleted
-    )
+        ),
+        identifiers = List(
+          SourceIdentifier(
+            identifierScheme = IdentifierSchemes.sierraSystemNumber,
+            sierraItemData.id
+          )
+        ),
+        locations = getLocation(sierraItemData).toList
+      ))
+    }
   }
 
   def getItems(
     sierraTransformable: SierraTransformable): List[UnidentifiedItem] = {
     extractItemData(sierraTransformable)
```

Comment:
What if we `.filter` on `deleted` here?

Reply:
Yeah, actually, that's simpler, isn't it?

Reply:
👍